src/main/resources/META-INF/persistence.xml

```diff
@@ -1,7 +1,7 @@
 <persistence xmlns="http://java.sun.com/xml/ns/persistence"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-   xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_1.xsd"
-   version="2.1">
+   xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd"
```

Comment:
looks like we are using 2.1 xsd and we just need to update the links

see http://www.oracle.com/webfolder/technetwork/jsc/xml/ns/persistence/index.html#2.1